crates/ansi-to-html/fuzz/utils/html_interpreter.rs

```diff
@@ -190,7 +190,7 @@ impl StylizedText {
 pub struct Styles {
     bold: bool,
     italic: bool,
-    underlined: bool,
+    underlined: Option<Option<Attr>>,
```

Comment:
I don't like nested `Option`s. Can we use the `UnderlineStyle` enum here?

Reply:
Done and done :tada:

I copied the definition out, but alternatively the original definition could be exposed publicly through a lot of `cfg(fuzzing)`